composer.json

```diff
@@ -15,7 +15,16 @@
         }
     ],
     "require": {
-        "php": ">=7.0 <=7.3",
+        "php": ">=7.0 <=7.1",
```

Comment:
I'm wondering if this should be < 7.2 to include all the minor versions of 7.1 ?

Reply:
You are very right :+1: and there's other stuff I missed . Preparing a new commit

Reply:
Fixed